client/src/pages/index.tsx

```diff
@@ -37,7 +37,22 @@ export default function Home() {
         fontSans.variable,
       )}
     >
-      <h1 className="text-3xl text-primary">Test title</h1>
+      <h1 className="title1 text-primary">Test title</h1>
+      {/* Test for font */}
+      {/*
+      <h1 className="largetitle text-primary">Largetitle</h1>
```

Comment:
please create a new page to show all the use-cases and delete these comments

Reply:
Completed modification in creating `font` page in `font.tsx`
To access it plz use `localhost:3000/font`